godotenv.go

```diff
@@ -28,6 +30,14 @@ import (
 
 const doubleQuoteSpecialChars = "\\\n\r\"!$`"
 
+func stringSet(s []string) map[string]interface{} {
+	m := make(map[string]interface{})
+	for _, str := range s {
+		m[str] = true
+	}
+	return m
+}
```

Comment:
Could you change a little bit the method? `map[string]struct{}` is a common way for implementing `set` in Go:
```suggestion
func stringSet(s []string) map[string]struct{} {
	m := make(map[string]struct{}, len(s))
	for _, str := range s {
		m[str] = struct{}{}
	}
	return m
}
```